docs/Quick-Start.md

```diff
@@ -89,12 +91,12 @@ EOF
 
 ```
 val dataFilePath = new File("").getCanonicalPath + "/carbondata/sample.csv"
-cc.sql(s"load data inpath '$dataFilePath' into table table1")
+cc.sql(s"load data local inpath '$dataFilePath' into table table1")
```

Comment:
Right now there is no significant meaning to `local` in carbondata.We just keep it to align  with hive syntax. It confues user so better don't use here.
 Here also you can give Note as `dataFilePath` can be hdfs path as well like 
`val dataFilePath = hdfs://hacluster//carbondata/sample.csv`